schema.sql

```diff
@@ -7,3 +7,12 @@ CREATE TABLE animals (
 INT NOT NULL, neutered BOOL NOT NULL,weight_kg DECIMAL NOT NULL);
 
  ALTER TABLE Animals ADD COLUMN SPECIES TEXT ;
```

Comment:
- [ ] You have two CREATE TABLE ANIMALS statements in this file, kindly delete the first one since it only has the name attribute on it. If you don't delete it, you will get this error wen running this file:

![image](https://user-images.githubusercontent.com/77106962/207693415-c332290e-c5d4-4d64-bec7-b95421512271.png)